multiqc/modules/hops/hops.py

```diff
@@ -0,0 +1,131 @@
+        info="is an ancient DNA characteristics screening tool of output from the metagenomic aligner MALT.")
+
+        # Find and load any HOPS post-processing JSONs
+        self.hops_data = dict()
+
+        for f in self.find_log_files('hops',filehandles=True):
+            try:
+                self.parseJSON(f)
+            except KeyError:
+                logging.warning("Error loading file {}".format(f['fn']))
+
+        self.hops_data = self.ignore_samples(self.hops_data)
+
+        if len(self.hops_data) == 0:
+            raise UserWarning
+
+        log.info("Found {} samples".format(len(self.hops_data)))
+
+        # This type of data isn't 'summarise-able' for general stats, so 
+        # skipping straight to heatmap. We also won't write data file to the
+        # multiqc_data directory because it would be exactly same as input JSON. 
+        self.hops_heatmap()
+
+    def parseJSON(self, f):
+        """ Parse the JSON output from HOPS and save the summary statistics """
+
+        try:
+            parsed_json = json.load(f['f'])
+        except JSONDecodeError as e:
+            log.debug("Could not parse HOPS JSON: '{}'".format(f['fn']))
+            log.debug(e)
+            return None
+        
+        # Convert JSON to dict for easier manipulation
+        for s in parsed_json: 
+            s_name = self.clean_s_name(s, f['root'])
+            if s_name in self.hops_data: 
+                log.debug("Duplicate sample name found! Overwriting: {}".format(s_name))
+            self.add_data_source(f, s_name=s_name)
+            self.hops_data[s_name] = {}
+            for t in parsed_json[s]:
+                self.hops_data[s_name][t] = parsed_json[s][t][0]
+    
+    def hops_heatmap(self):
+        """ Heatmap showing all statuses for every sample """
+        heatmap_numbers = {
+            'none': 1,
+            'edit_only': 2,
+            'damage_only': 3,
+            'edit_and_damage': 4
+        }
+
+        samples = []
+        for s in self.hops_data:
+            samples.append(s)
+
+        # As all samples always have same taxa, will take from the first sample
+        taxa = []
+        for t in self.hops_data[samples[0]]:
+            taxa.append(t)
+
+        # Get values from named list into a list of lists required for heatmap
+        levels = []
+        for s in samples:
+            levels.append(self.hops_data[s].values())
+
+        pconfig = {
+            'id': 'hops-heatmap',
+            'title': 'HOPS: Potential Candidates',
+            'xTitle': 'Node',
+            'yTitle': 'Sample',
+            'min': 0,
+            'max': 1,
+            'square': False,
+            'colstops': [
+                [1, '#ffffff'],
+                [2, '#ffff33'],
+                [3, '#ff7f00'],
+                [4, '#e41a1c'],
+            ],
+            'decimalPlaces': 0,
+            'legend': False,
+            'datalabels': False,
+            'ycats_samples': True,
```

Comment:
`True` is the default..
```suggestion
            'xcats_samples': False,
```